wpengine/atlas-content-modeler
# [NOJIRA] Move author meta box to sidebar for ACM post types

## Description

Moves the Author meta box provided by WP Core from the main editor area to the sidebar.

Props to @mindctrl for suggesting this usability improvement. If we don't do this, the meta box can load before the React app, making publishers read that before the main app appears.

Alternatives are to delay the appearance of the meta box until the editor app has loaded, or hide the author meta box on the publisher screen.

## Testing

Amends an existing e2e test to check the meta box moved to the sidebar.

### Manual testing

Create a new model and add an entry. The Author meta box should appear in the sidebar at larger screen widths.

If you move any of the meta boxes using the up/down arrows during testing, you will need to reset the meta box user preferences for that post type before boxes appear in their default positions again. For example, with WP-CLI:

```sh
wp user meta delete [user-slug] meta-box-order_[post-type-slug]
```

## Screenshots

| Before | After
|-|-|
| <img width="1203" alt="Screenshot 2021-10-12 at 10 52 42" src="https://user-images.githubusercontent.com/647669/136924654-b9cd0cd4-bb98-487e-835b-fc270c7bff3d.png"> | <img width="1207" alt="Screenshot 2021-10-12 at 10 52 23" src="https://user-images.githubusercontent.com/647669/136924697-30c7b5d1-c4ad-43c3-9099-d195017f64ad.png"> |

## Documentation Changes

n/a

## Dependant PRs

n/a
